tests/verify_membackend.erl

```diff
@@ -152,7 +152,10 @@ check_put_delete(Node) ->
     Mem = get_used_space(Pid, Node),
 
     %% this is meh, but the value isn't always the same length.
-    case (Mem == MemBaseline - 1142) orelse 
+    %% It seems to be the size of a Riak Object
+    case (Mem == MemBaseline - 1146) orelse
```

Comment:
Not sure why the test was set up this way. I suspect it was the size of the Riak Object, but of course it can change over time.  I could set a range, but as soon as the size changes, the test will break again.